examples/fungible/src/contract.rs

```diff
@@ -143,6 +143,9 @@ impl FungibleTokenContract {
                     "The requested transfer is not correctly authenticated."
                 )
             }
+            AccountOwner::Chain => {
+                unreachable!("Chains cannot be used to authenticate")
```

Comment:
`panic!("Chain account is not supported")`

Reply:
We could totally support it by the way: it would authorized from operations and unauthorized from messages.

Reply:
Done in [Replace unreachables with panics](https://github.com/linera-io/linera-protocol/pull/3560/commits/f8d5e33ec6a0610a99a2b4a9524e399cf37e60c4).